src/step-registry.js

```diff
@@ -11,15 +11,20 @@ var StepRegistry = function () {
 StepRegistry.prototype.add = function (generalisedText, stepText, stepFunction, filePath, line, options) {
   if (this.exists(generalisedText)) {
     this.registry[generalisedText].count++;
+    this.registry[generalisedText].fileLocations.push({ filePath: filePath, line: line });
     return;
   }
 
   this.registry[generalisedText] = {
     fn: stepFunction,
     stepText: stepText,
     generalisedText: generalisedText,
-    filePath: filePath,
-    line: line,
+    fileLocations: [
+      {
+        filePath: filePath,
+        line: line
+      }
+    ],
     count: 1,
```

Comment:
Why is this required? It could be a method which returns `fileLocations.length`.